packages/devtools/lib/src/timeline/frame_flame_chart.dart

```diff
@@ -25,17 +25,17 @@ bool _debugEventTrace = false;
 // Blue 100-300 (light mode) or 400, 600, 700 (dark mode) color palette from
 // https://material.io/design/color/the-color-system.html#tools-for-picking-colors.
 final cpuColorPalette = [
-  isDarkTheme ? mainCpuColor : const Color(0xFFBBDEFB),
-  isDarkTheme ? const Color(0xFF1E88E5) : const Color(0xFF90CAF9),
-  isDarkTheme ? const Color(0xFF1976D2) : mainCpuColor,
+  const ThemedColor(Color(0xFFBBDEFB), mainCpuColor),
+  const ThemedColor(Color(0xFF90CAF9), Color(0xFF1E88E5)),
+  const ThemedColor(mainCpuColor, Color(0xFF1976D2)),
```

Comment:
why isn't mainCpuColor a themed color?

Reply:
it is. It is defined in timeline.dart because it is also used by the frame rendering chart.